sideboard/tests/__init__.py

```diff
@@ -31,7 +31,8 @@ def patch_config(value, *path, **kwargs):
         conf = kwargs.pop('config', config)
         for section in path[:-1]:
             conf = conf[section]
-        request.addfinalizer(lambda: conf.__setitem__(path[-1], value))
+        orig_val = conf[path[-1]]
+        request.addfinalizer(lambda: conf.__setitem__(path[-1], orig_val))
```

Comment:
FWIW, this whole function is deprecated, since ``monkeypatch.setitem`` in ``pytest`` is a better and more Pythonic way to do this anyway.  This whole function predates our use of ``pytest``.